docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+
+    Manual Arming Throttle Source Parameter in IQ Control Center
+
+When the module is armed manually, it will use the *Manual Arming Throttle Source* as its armed throttle source. **That means that it will only listen to throttle 
+commands from the specified source for spinning and disarming when it is armed manually.** For example, if you had the *Manual Arming Throttle Source* set to *DroneCAN* and manually armed the module,
+then any throttle commands from DroneCAN would be accepted, but any sent over :ref:`hobby_standard_pwm` would be rejected. **It is essential to set the** *Manual Arming Throttle Source*
+**before attempting to arm manually.**
+
+Timeout
+********
+Timeouts will always trigger a disarming transition. If the module is armed just before a timeout occurs, it will be disarmed just after the timeout. Note that 
+whether the module performs its disarming behavior or not on a timeout depends on the timeout feature configurations. See the :ref:`manual_timeout` section, specifically the 
+:ref:`timeout_meaning` subsection, for more information.
+
+Always Armed
+*************
+For users who want to avoid any arming procedure altogether, there is the *Always Armed* parameter. **This parameter allows users to specify that they want the motor to 
+immediately arm on the first throttle message it receives, regardless of its value.** This parameter can be found in the Advanced tab of IQ Control Center as shown below.
+
+.. figure:: ../_static/manual_images/arming/always_armed_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Always Armed Parameter
+
+    Always Armed Parameter in IQ Control Center
+
+**When this parameter is set to the** *Always Armed* **value, the first throttle command received will cause the module to immediately arm.** The module will then set 
+its armed throttle source to whatever is specified in the *Manual Arming Throttle Source* parameter. For more information on the *Manual Arming Throttle Source* parameter, 
+see :ref:`manual_arming_throttle_source`. **If using** *Always Armed*, **it is essential to also set the** *Manual Arming Throttle Source* **appropriately so the proper source 
+will be used by the module after arming.**
+
+**It is recommended to avoid using this feature and to leave this parameter set to** *Normal Arming* **whenever possible.** Arming on any throttle message may lead 
+to the module arming on high percentage throttles and suddenly spinning rapidly, posing a risk to anyone nearby. Using arming throttle regions as described 
+in :ref:`throttle_regions` is generally safer, as the module can be set to only arm on low percentage throttles, mitigating the risk that an unexpected arm might pose.
+
+.. _advanced_arming_behavior:
+
+Arming Behavior
+================
+When an arming transition (from disarmed to armed) occurs, the module executes its arming behavior. The arming behavior is not currently configurable, it will always be the same. 

```

Comment:
Why even mention that it's not configurable. Just say that it executes the arming behavior described below

Reply:
Just as a contrast to disarming behavior. If I looked at disarming behavior with all its configurations and then looked at arming behavior, I'd wonder what the configuration options for arming behavior are.